hack/lib/golang.sh

```diff
@@ -100,7 +100,6 @@ else
     linux/amd64
     darwin/amd64
     windows/amd64
-    linux/arm
```

Comment:
Yes, when things are even more stable, when I've had more time to dig into these cherrypicks, got more help from the Go team, and when we're using go1.7.1 at master.

I'm doing this to reduce the risk; @pwittrock and everyone wants as low risk as possible.

The priority is getting the apiserver working